src/Codeception/Lib/Connector/Yii2.php

```diff
@@ -252,6 +251,6 @@ protected function persistDb()
 
     private function mockAssetManager()
     {
-        $this->app->set('assetManager', Stub::make('yii\web\AssetManager'));
+        $this->app->set('assetManager', new Maybe());
```

Comment:
I think you approach is better because Maybe class is pretty flexible, however once anywhere an application `AssetManager`'s class or interface will be checked a test will fail.